### ISSUE TITLE
Tutorial for using FID metric?

### ISSUE THREAD

**Issue by zzeitt** [photosynthesis-team/piq#241]
**Is your feature request related to a problem? Please describe.**
I don't know how to measure FID score for two single images.
The example codes about FID is not  runnable.

**Describe the solution you'd like**
I want a complete codes showing how to use FID metric to measure FID between two images.

**Describe alternatives you've considered**
Maybe some examples of making use of FID on torch's DataLoader object.

**Additional context**
(Null)

**Comment by zakajd** [comment]
Hi @zzeitt 
FID metrics is used to measure distance between two _image distributions_, not individual images. 
We didn't provide the exact code because Dataloader implementations can be task-specific.

**Comment by zzeitt** [comment]
Thanks for your quick reply as well as your explanations on FID metric. I am able to measure the FID between two sets of images now.
And for those who is as confused as me of applying FID metric, I will post my runnable codes here:
``` python
import os
import piq
import torch
import torchvision.transforms as transforms
from PIL import Image


def getFilePaths(path):
    # read a folder, return the complete path
    ret = []
    for root, dirs, files in os.walk(path):
        for filespath in files:
            ret.append(os.path.join(root, filespath))
    ret.sort()
    return ret


class DataProcess(torch.utils.data.Dataset):
    def __init__(self, s_src, img_w, img_h):
        super(DataProcess, self).__init__()
        self.img_w = img_w
        self.img_h = img_h

        self.img_transform = transforms.Compose([
            transforms.Resize((self.img_h, self.img_w)),
            transforms.ToTensor(),  # transform to [0, 1]
        ])

        self.f_srcs = getFilePaths(s_src)

    def __getitem__(self, index):
        src = Image.open(self.f_srcs[index])
        t_src = self.img_transform(src.convert('RGB'))

        return {
            'images': t_src,
        }

    def __len__(self):
        return len(self.f_srcs)


if __name__ == '__main__':   
    s_1 = ('./variables/1-in-src/dataset/forDemo/text')
    s_2 = ('./variables/1-in-src/dataset/forDemo/misc')
    
    set_1 = DataProcess(s_1, 256, 256)
    set_2 = DataProcess(s_2, 256, 256)
    
    loader_1 = torch.utils.data.DataLoader(set_1, batch_size=1, shuffle=False)
    loader_2 = torch.utils.data.DataLoader(set_2, batch_size=1, shuffle=False)
    
    fid_metric = piq.FID()
    feat_1 = fid_metric._compute_feats(loader_1)
    feat_2 = fid_metric._compute_feats(loader_2)
    fid = fid_metric.compute_metric(feat_1, feat_2)
    print(f'====> fid: {fid}')
    
```

Again, thanks for your good work!
